server/interpreter.js

```diff
@@ -2744,12 +2747,12 @@ Interpreter.prototype['stepMemberExpression'] = function() {
     stack.push({node: node['property'], components: !node['computed']});
```

Comment:
As noted: if `!computed`, `node.property` will be of type `Identifer` and you can just set `state.propName = node.property.name`.